### PR TITLE
feat(affiliates): add pr template generation and app

### LINKED ISSUE #2657: add pr template for dev mining automation
**Why**

During the automated rewards generation for dev mining, we want to be able to open a github PR. For scripting purposes we want to output a string with a body and title which can be used to do this. 

**How**

Add a dev mining pr template function to affiliates utils and an app which will take in a config and output {title,body}

**Resolution**

See that you can run dev mining and output the template strings to console.

**Difficulty Score [1-10]**

- 4: ~2 hours

### PR DESCRIPTION
**Pull request by daywiss**
Signed-off-by: David Adams <david@umaproject.org>

**Motivation**
#2657

**Summary**

Updates the dev mining pr string, and documents how to use the app to generate pr message.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [x]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

Fixes #2657

### CODE REVIEW

File: packages/affiliates/libs/affiliates/utils.js [hunk 1]
```diff
@@ -75,14 +76,14 @@ function devMiningPrTemplate({
     .utc()
     .format("YYYY/MM/DD");
   return {
-    title: `Run Dev Mining rewards for week ${weekNumber + 1} between ${startDate} and ${endDate}`,
+    title: `improve(affiliates): Dev Mining rewards for week ${weekNumber}`,
```
Comment (nicholaspai) [inline packages/affiliates/libs/affiliates/utils.js:79]: +1

**Review by nicholaspai — APPROVED** [review]
+1 adding conventional commit format

File: packages/affiliates/apps/GenerateDevMiningPr.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+require("dotenv").config();
+const { makeUnixPipe, devMiningPrTemplate } = require("../libs/affiliates/utils");
+
+// This expects to be piped json input which contains this structure:
+// {
+//   "issueNumber": 2620,
```
Comment (chrismaree) [inline packages/affiliates/apps/GenerateDevMiningPr.js:6]: where does the issue number come from?

**Review by chrismaree — APPROVED** [review]

File: packages/affiliates/apps/GenerateDevMiningPr.js (hunk 2 above)
Comment (daywiss) [reply packages/affiliates/apps/GenerateDevMiningPr.js:6]: from [apps/OpenDevMiningIssue](https://github.com/UMAprotocol/protocol/blob/master/packages/affiliates/apps/OpenDevMiningIssue.js#L23)

**Pull request merged**